### DISCUSSION TITLE
Is there an actual dependency on Fragpipe-DIA

### DISCUSSION THREAD

**Discussion by sooheon** [Nesvilab/FragPipe-Analyst-doc#50] [Ideas]
The [quickstart](http://fragpipe-analyst.nesvilab.org) mentions one should use output of Fragpipe-DIA as input to the analyst

> For protein: Upload protein group (PG) matrix ( diann-output.pg_matrix.tsv ) generated by DIA-NN in FragPipe
> For peptide: Upload precursor group matrix ( diann-output.pr_matrix.tsv ) generated by DIA-NN in FragPipe

But I can also use DIA data searched and quantified by DIA-NN directly, correct? I assume there is no additional column that FragpipeDIA adds that this tool depends on.

**Comment by hsiaoyi0504** [comment]
I think there is still some difference like DIA-NN result might contain multiple proteins reported, see https://github.com/Nesvilab/FragPipeAnalystR/issues/14.
